hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/action/cluster/strategy/BaseConsistentHashingBucketClusteringPlanStrategy.java

```diff
@@ -128,7 +129,7 @@ protected Stream<HoodieClusteringGroup> buildClusteringGroupsForPartition(String
             .build();
       }).collect(Collectors.toList()));
     }
-    return ret.stream();
+    return Pair.of(ret.stream(), true);
```

Comment:
In which case the flag is false.

Reply:
This boolean value is used to indicate whether all candidate fileslice under the current partition have been processed. The external world uses this as a basis to determine whether the current partition is added to missingPartitions.

For `BaseConsistentHashingBucketClusteringPlanStrategy` it always return true.

For other `PartitionAwareClusteringPlanStrategy` like `SparkSizeBasedClusteringPlanStrategy` it may return false.
Such as , 100 fileSlices are passed in, but due to the limitation of `writeConfig.getClusteringMaxNumGroups()`, only 10 of them are processed. In this case, false should be returned.